test/e2e/bundle_image_test.go

```diff
@@ -3,6 +3,8 @@ package e2e_test
 import (
 	"context"
 	"fmt"
+	"github.com/onsi/ginkgo/extensions/table"
```

Comment:
nit: looks like the ordering of the new dependencies is causing goimports to fail so CI is red
```
find . -name '*.go' -not -path "./vendor/*" | xargs goimports -w
diff --git a/test/e2e/bundle_image_test.go b/test/e2e/bundle_image_test.go
index 637795d..5615346 100644
--- a/test/e2e/bundle_image_test.go
+++ b/test/e2e/bundle_image_test.go
```

Reply:
Also, squashing all the commits for the final PR would be great as well

Reply:
Fixed both.